File: app/src/main/java/com/juniperphoton/myersplash/utils/DisplayUtil.java
package com.juniperphoton.myersplash.utils;

import android.content.Context;

public class DisplayUtil {

    public static float getDpi(Context context) {
        return context.getResources().getDisplayMetrics().density;
    }

    public static int getDimenInPixel(int valueInDP, Context context) {
        return (int) (valueInDP * getDpi(context));
    }
}

File: app/src/main/java/com/juniperphoton/myersplash/utils/DisplayUtil.kt
package com.juniperphoton.myersplash.utils

import android.content.Context

class DisplayUtil {
    companion object {
        fun getDpi(context: Context): Float {
            return context.resources.displayMetrics.density
        }

        fun getDimenInPixel(valueInDP: Int, context: Context): Int {
            return (valueInDP * getDpi(context)).toInt()
        }
    }
}
